src/day_05/iterableAndIterators.test.js

```diff
@@ -0,0 +1,50 @@
+
+describe('DAY 5: Iterable/Iterator', () => {
+
+    it(`get the iterator function of a String`, () => {
+        let string = 'hello';
+        let theIteratorFunction;
+
+        expect(theIteratorFunction).toBeInstanceOf(Object);
```

Comment:
no function should be created here, "get the iterator function" -> "capture the iteration function already present on every string"
```javascript
let theIteratorFunction = string[Symbol.iterator]() // this will return the StringIterator which is an iterator object
```